onboard/src/components/Accordion.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import { Divider } from "@mui/material";
+import Box from "@mui/material/Box";
+import { Button } from "@mui/material";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import ArrowDropUpIcon from "@mui/icons-material/ArrowDropUp";
+
+// Flexible Accordion module with a number of props, some of which are optional (importance, open)
```

Comment:
We can remove the part of the comment that mentions 'importance' as it's not a feature we are using. My bad in sending that comment unaltered.

Reply:
I can do this